tests/test_rio_merge.py

```diff
@@ -736,7 +733,8 @@ def test_merge_no_gap(tiffs, runner):
 
     with rasterio.open(outputname) as src:
         data = src.read(1)
-        assert data[184, 61] != 0
+        assert data[183, 61] != 0
+        assert data[184, 60] != 0
```

Comment:
SImilar story here. The gap we're testing for remains out of the picture, but the merged raster has shifted.